### PR TITLE
Adding German Language Support

### PR DESCRIPTION
**Pull request by JustZerooo**
- German Language Support in Core Modules.

### CODE REVIEW

**Comment by zRxnx** [comment]
Thats not a good translations, looks like an ai or smth like that

**Comment by JustZerooo** [comment]
It is the german translation how we speak that, i am german, how i am an ai ... lol

**Comment by Kenshiin13** [comment]
> Thats not a good translations, looks like an ai or smth like that

Translations LGTM.

File: [core]/esx_identity/locales/de.lua [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Locales["de"] = {
+  ["show_active_character"] = "Aktiven Charakter anzeigen",
+  ["active_character"] = "Aktiver Charakter: %s",
+  ["error_active_character"] = "Beim Abrufen Ihrer Daten ist ein Fehler aufgetreten",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:4]:
```suggestion
  ["error_active_character"] = "Beim Abrufen deiner Daten ist ein Fehler aufgetreten",
```

File: [core]/esx_identity/locales/de.lua [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  ["delete_character"] = "Lösche deinen aktuellen Charakter.",
+  ["deleted_character"] = "Charakter gelöscht",
+  ["error_delete_character"] = "Es gab ein Problem beim löschen deines Charakters.",
+  ["thank_you_for_registering"] = "Registrierung erfolgreich. Enjoy!",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:8]:
```suggestion
  ["thank_you_for_registering"] = "Registrierung erfolgreich.",
```

File: [core]/esx_identity/locales/de.lua [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  ["debug_xPlayer_get_first_name"] = "Gibt Ihren Vornamen zurück",
+  ["debug_xPlayer_get_last_name"] = "Gibt Ihren Nachnamen zurück",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:10]:
```suggestion
  ["debug_xPlayer_get_last_name"] = "Gibt deinen Nachnamen zurück",
```

File: [core]/esx_identity/locales/de.lua [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  ["debug_xPlayer_get_full_name"] = "Gibt Ihren vollständigen Namen zurück",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:11]:
```suggestion
  ["debug_xPlayer_get_full_name"] = "Gibt deinen vollständigen Namen zurück",
```

File: [core]/esx_identity/locales/de.lua [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  ["debug_xPlayer_get_sex"] = "Gibt dein Geschlecht zurück",
+  ["debug_xPlayer_get_dob"] = "Gibt dein Geburtsdatum zurück",
+  ["debug_xPlayer_get_height"] = "Gibt deine Größe zurück",
+  ["error_debug_xPlayer_get_first_name"] = "Es gab ein Problem beim Abrufen deines Vornamens.",
+  ["error_debug_xPlayer_get_last_name"] = "Es gab ein Problem beim Ermitteln deines Nachnamens.",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:16]:
```suggestion
  ["error_debug_xPlayer_get_last_name"] = "Es gab ein Problem beim Abrufen deines Nachnamens.",
```

File: [core]/esx_identity/locales/de.lua [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  ["error_debug_xPlayer_get_full_name"] = "Es gab ein Problem beim Abrufen deines vollständigen Namens.",
+  ["error_debug_xPlayer_get_sex"] = "Es gab ein Problem bei der Ermittlung deines Geschlechts.",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:18]:
```suggestion
  ["error_debug_xPlayer_get_sex"] = "Es gab ein Problem beim Abrufen deines Geschlechts.",
```

File: [core]/esx_identity/locales/de.lua [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  ["error_debug_xPlayer_get_dob"] = "Es gab ein Problem bei der Ermittlung deines Geburtsdatums.",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:19]:
```suggestion
  ["error_debug_xPlayer_get_dob"] = "Es gab ein Problem beim Abrufen deines Geburtsdatums.",
```

File: [core]/esx_identity/locales/de.lua [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+  ["error_debug_xPlayer_get_height"] = "Es gab ein Problem bei der Ermittlung Ihrer Größe.",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:20]:
```suggestion
  ["error_debug_xPlayer_get_height"] = "Es gab ein Problem beim Abrufen deiner Körpergröße.",
```

File: [core]/esx_identity/locales/de.lua [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  ["return_debug_xPlayer_get_first_name"] = "Vorname: %s",
+  ["return_debug_xPlayer_get_last_name"] = "Nachname: %s",
+  ["return_debug_xPlayer_get_full_name"] = "Name: %s",
+  ["return_debug_xPlayer_get_sex"] = "Geschlecht: %s",
+  ["return_debug_xPlayer_get_dob"] = "Geburtstag: %s",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:25]:
```suggestion
  ["return_debug_xPlayer_get_dob"] = "Geburtsdatum: %s",
```

File: [core]/esx_identity/locales/de.lua [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+  ["return_debug_xPlayer_get_height"] = "Größe: %s CM",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:26]:
```suggestion
  ["return_debug_xPlayer_get_height"] = "Größe: %s cm",
```

File: [core]/esx_identity/locales/de.lua [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+  ["data_incorrect"] = "Ungültige Daten, bitte versuche es erneut",
+  ["invalid_format"] = "Ungültiges Datenformat, bitte versuche es erneut.",
+  ["no_identifier"] = "Es gab ein Problem beim Laden deines Charakters!\nFehlercode: identifier-missing\n\nDies wird dadurch verursacht, dass dein Bezeichner fehlt. Bitte komme später wieder oder melde dieses Problem dem Support.",
+  ["missing_identity"] = "Es gab ein Problem beim Laden deines Charakters!\nFehlercode: identity-missing\n\nEs scheint, dass deine Identität fehlt, versuche erneut, eine Verbindung herzustellen.",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:30]:
```suggestion
  ["missing_identity"] = "Es gab ein Problem beim Laden deines Charakters!\nFehlercode: identifier-missing\n\nDies wird dadurch verursacht, dass deine R* ID fehlt. Bitte versuche es erneut oder melde dieses Problem dem Support.",
```

File: [core]/esx_identity/locales/de.lua [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+  ["deleted_identity"] = "Charakter gelöscht. Bitte verbindedich erneut, um einen neuen Charakter zu erstellen.",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:31]:
```suggestion
  ["deleted_identity"] = "Charakter gelöscht. Bitte verbinde dich erneut, um einen neuen Charakter zu erstellen.",
```

File: [core]/esx_identity/locales/de.lua [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+  ["already_registered"] = "Du hast dich bereits registriert.",
+  ["invalid_firstname_format"] = "Ungültiges Format (Vorname): Bitte versuche es erneut.",
+  ["invalid_lastname_format"] = "Ungültiges Format (Nachname): Bitte versuche es noch einmal.",
+  ["invalid_dob_format"] = "Ungültiges Format (Geburtstag): Bitte versuche es noch einmal.",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:35]:
```suggestion
  ["invalid_dob_format"] = "Ungültiges Format (Geburtstag): Bitte versuche es erneut.",
```

File: [core]/esx_identity/locales/de.lua [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+  ["invalid_sex_format"] = "Ungültiges Format (Geschlecht): Bitte versuche es erneut.",
+  ["invalid_height_format"] = "Ungültiges Format (Größe): Bitte versuche es erneut.",
```
Comment (zRxnx) [inline [core]/esx_identity/locales/de.lua:37]:
```suggestion
  ["invalid_height_format"] = "Ungültiges Format (Körpergröße): Bitte versuche es erneut.",
```

**Comment by Kenshiin13** [comment]
@zRxnx Thanks for the spamm, my phone is on fire now.

File: [core]/esx_multicharacter/locales/de.lua [hunk 15]
```diff
@@ -0,0 +1,32 @@
+Locales["de"] = {
+	[" male"] = "Männlich",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:2]:
```suggestion
	["male"] = "Männlich",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+	[" female"] = "Weiblich",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:3]:
```suggestion
	["female"] = "Weiblich",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+	["select_char"] = "Charakter Auswählen",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:4]:
```suggestion
	["select_char"] = "Charakter auswählen",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 18]
```diff
@@ -0,0 +1,32 @@
...
+	["select_char_description"] = "Wähle einen Charakter aus, als den du spielen willst.",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:5]:
```suggestion
	["select_char_description"] = "Wähle einen Charakter aus, mit den du spielen willst.",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 19]
```diff
@@ -0,0 +1,32 @@
...
+	["create_char"] = "Neuer Charakter",
+	["char_play"] = "Spielen",
+	["char_play_description"] = "Weiter in die Stadt.",
+	["char_disabled"] = "Deaktiviert",
+	["char_disabled_description"] = "Dieser Charakter ist unbrauchbar.",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:10]:
```suggestion
	["char_disabled_description"] = "Dieser Charakter ist deaktiviert.",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+	["char_delete"] = "Löschen",
+    ["char_delete_description"] = "Dieser Charakter wird dauerhaft entfernt.",
+	["char_delete_confirmation"] = "Löschbestätigung",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:13]:
```suggestion
	["char_delete_confirmation"] = "Bestätigung zur Entfernung deines Charakters.",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 21]
```diff
@@ -0,0 +1,32 @@
...
+	["char_delete_confirmation_description"] = "Bist du sicher, dass du den ausgewählten Charakter entfernen willst?",
+	["char_delete_yes_description"] = "Ja, ich bin sicher, dass ich den ausgewählten Charakter lösche",
+	["char_delete_no_description"] = "Nein, zurück zu den Charakteroptionen",
+	["character"] = " Charakter: %s",
+	["return"] = "Zurück",
+	["return_description"] = "Zur Charakterauswahl zurückkehren.",
+	["command_setslots"] = "Anzahl der Multicharakter-Slots eines Spielers festlegen",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:20]:
```suggestion
	["command_setslots"] = "Anzahl der Charakter-Slots eines Spielers festlegen",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 22]
```diff
@@ -0,0 +1,32 @@
...
+	["command_remslots"] = "Die Anzahl der Multicharakter-Slots eines Spielers entfernen",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:21]:
```suggestion
	["command_remslots"] = "Die Anzahl der Charakter-Slots eines Spielers entfernen",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 23]
```diff
@@ -0,0 +1,32 @@
...
+	["command_enablechar"] = "Aktiviere einen bestimmten Charakter eines Spielers",
+	["command_disablechar"] = "Einen bestimmten Charakter eines Spielers deaktivieren",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:23]:
```suggestion
	["command_disablechar"] = "Deaktiviere einen bestimmten Charakter eines Spielers",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 24]
```diff
@@ -0,0 +1,32 @@
...
+	["command_charslot"] = "Slotnummer des Charakters",
+	["command_identifier"] = "Spieler-Identifikator",
+	["command_slots"] = "# of slots",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:26]:
```suggestion
	["command_slots"] = "Anzahl von Charakter-Slots",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 25]
```diff
@@ -0,0 +1,32 @@
...
+	["char_delete_no_description"] = "Nein, zurück zu den Charakteroptionen",
+	["character"] = " Charakter: %s",
+	["return"] = "Zurück",
+	["return_description"] = "Zur Charakterauswahl zurückkehren.",
+	["command_setslots"] = "Anzahl der Multicharakter-Slots eines Spielers festlegen",
+	["command_remslots"] = "Die Anzahl der Multicharakter-Slots eines Spielers entfernen",
+	["command_enablechar"] = "Aktiviere einen bestimmten Charakter eines Spielers",
+	["command_disablechar"] = "Einen bestimmten Charakter eines Spielers deaktivieren",
+	["command_charslot"] = "Slotnummer des Charakters",
+	["command_identifier"] = "Spieler-Identifikator",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:25]:
```suggestion
	["command_identifier"] = "Spieler R* ID",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 26]
```diff
@@ -0,0 +1,32 @@
...
+	["command_slots"] = "# of slots",
+	["slotsadd"] = "Du hast %s Slots auf %s gesetzt",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:27]:
```suggestion
	["slotsadd"] = "Du hast %s Charakter-Slots auf %s gesetzt",
```

File: [core]/esx_multicharacter/locales/de.lua [hunk 27]
```diff
@@ -0,0 +1,32 @@
...
+	["slotsrem"] = "Du hast Slots auf %s entfernt",
```
Comment (zRxnx) [inline [core]/esx_multicharacter/locales/de.lua:28]:
```suggestion
	["slotsrem"] = "Du hast &s Charakter-Slots auf %s entfernt",
```

File: [core]/esx_skin/locales/de.lua [hunk 28]
```diff
@@ -0,0 +1,6 @@
+Locales["de"] = {
+    ["skin_menu"] = "Skin Menu",
```
Comment (zRxnx) [inline [core]/esx_skin/locales/de.lua:2]:
```suggestion
    ["skin_menu"] = "Skin Menü",
```

File: [core]/esx_skin/locales/de.lua [hunk 29]
```diff
@@ -0,0 +1,6 @@
...
+    ["use_rotate_view"] = "Verwende ~INPUT_FRONTEND_LS~ und ~INPUT_CHARACTER_WHEEL~, um die Ansicht zu drehen.",
```
Comment (zRxnx) [inline [core]/esx_skin/locales/de.lua:3]:
```suggestion
    ["use_rotate_view"] = "Verwende ~INPUT_FRONTEND_LS~ und ~INPUT_CHARACTER_WHEEL~ um deine Ansicht zu ändern.",
```

File: [core]/esx_skin/locales/de.lua [hunk 30]
```diff
@@ -0,0 +1,6 @@
...
+    ["skin"] = "Skin ändern",
```
Comment (zRxnx) [inline [core]/esx_skin/locales/de.lua:4]:
```suggestion
    ["skin"] = "Aussehen ändern",
```

File: [core]/esx_skin/locales/de.lua [hunk 31]
```diff
@@ -0,0 +1,6 @@
...
+    ["saveskin"] = "Skin in einer Datei speichern",
```
Comment (zRxnx) [inline [core]/esx_skin/locales/de.lua:5]:
```suggestion
    ["saveskin"] = "Aussehen abspeichern",
```

File: [core]/skinchanger/locales/de.lua [hunk 32]
```diff
@@ -0,0 +1,100 @@
...
+  ["nose_1"] = "Nasenbreite",
+  ["nose_2"] = "Höhe der Nasenspitze",
+  ["nose_3"] = "Länge der Nasenspitze",
+  ["nose_4"] = "Nasenbeinhöhe",
+  ["nose_5"] = "Nasenspitzenabsenkung",
+  ["nose_6"] = "Nasenbeinverdrehung",
+  ["cheeks_1"] = "Wangenknochenhöhe",
+  ["cheeks_2"] = "Wangenknochenbreite",
+  ["cheeks_3"] = "Wangenbreite",
+  ["lip_fullness"] = "Lippenfülle",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:16]:
```suggestion
  ["lip_fullness"] = "Lippendicke",
```

File: [core]/skinchanger/locales/de.lua [hunk 33]
```diff
@@ -0,0 +1,100 @@
...
+  ["chin_width"] = "Kinnbreite",
+  ["chin_hole"] = "Kinnlochgröße",
+  ["neck_thickness"] = "Halsdicke",
+  ["wrinkles"] = "Falten",
+  ["wrinkle_thickness"] = "Faltenstärke",
+  ["beard_type"] = "Barttyp",
+  ["beard_size"] = "Bartgröße",
+  ["beard_color_1"] = "Bartfarbe 1",
+  ["beard_color_2"] = "Bartfarbe 2",
+  ["hair_1"] = "Haar 1",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:30]:
```suggestion
  ["hair_1"] = "Haare 1",
```

File: [core]/skinchanger/locales/de.lua [hunk 34]
```diff
@@ -0,0 +1,100 @@
...
+  ["hair_2"] = "Haar 2",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:31]:
```suggestion
  ["hair_2"] = "Haare 2",
```

File: [core]/skinchanger/locales/de.lua [hunk 35]
```diff
@@ -0,0 +1,100 @@
...
+  ["eye_color"] = "Augenfarbe",
+  ["eye_squint"] = "Augenblinzeln",
+  ["eyebrow_type"] = "Augenbrauentyp",
+  ["eyebrow_size"] = "Augenbrauengröße",
+  ["eyebrow_color_1"] = "Augenbrauenfarbe 1",
+  ["eyebrow_color_2"] = "Augenbrauenfarbe 2",
+  ["eyebrow_depth"] = "Augenbrauentiefe",
+  ["eyebrow_height"] = "Augenbrauenhöhe",
+  ["makeup_type"] = "Makeup-Typ",
+  ["makeup_thickness"] = "Makeup-Dicke",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:43]:
```suggestion
  ["makeup_thickness"] = "Makeupstärke",
```

File: [core]/skinchanger/locales/de.lua [hunk 36]
```diff
@@ -0,0 +1,100 @@
...
+  ["lipstick_color_2"] = "Lippenstiftfarbe 2",
+  ["ear_accessories"] = "Ohrenzubehör",
+  ["ear_accessories_color"] = "Ohrenzubehör Farbe",
+  ["tshirt_1"] = "T-Shirt 1",
+  ["tshirt_2"] = "T-Shirt 2",
+  ["torso_1"] = "Oberkörper 1",
+  ["torso_2"] = "Oberkörper 2",
+  ["decals_1"] = "Abziehbilder 1",
+  ["decals_2"] = "Abziehbilder 2",
+  ["arms"] = "Arme",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:58]:
```suggestion
  ["arms"] = "Arme 1",
```

File: [core]/skinchanger/locales/de.lua [hunk 37]
```diff
@@ -0,0 +1,100 @@
...
+  ["watches_1"] = "Uhren 1",
+  ["watches_2"] = "Uhren 2",
+  ["bracelets_1"] = "Armbänder 1",
+  ["bracelets_2"] = "Armbänder 2",
+  ["glasses_1"] = "Brille 1",
+  ["glasses_2"] = "Brille 2",
+  ["bag"] = "Tasche",
+  ["bag_color"] = "Taschenfarbe",
+  ["blemishes"] = "Schönheitsfehler",
+  ["blemishes_size"] = "Flecken-Dicke",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:81]:
```suggestion
  ["blemishes_size"] = "Fleckenstärke",
```

File: [core]/skinchanger/locales/de.lua [hunk 38]
```diff
@@ -0,0 +1,100 @@
...
+  ["ageing"] = "Alterung",
+  ["ageing_1"] = "Alterungsdicke",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:83]:
```suggestion
  ["ageing_1"] = "Alterungsstärke",
```

File: [core]/skinchanger/locales/de.lua [hunk 39]
```diff
@@ -0,0 +1,100 @@
...
+  ["blush"] = "Erröten",
+  ["blush_1"] = "Rouge-Dicke",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:85]:
```suggestion
  ["blush_1"] = "Errötungsstärke",
```

File: [core]/skinchanger/locales/de.lua [hunk 40]
```diff
@@ -0,0 +1,100 @@
...
+  ["blush_color"] = "Errötungsfarbe",
+  ["complexion"] = "Teint",
+  ["complexion_1"] = "Teintstärke",
+  ["sun"] = "Sonne",
+  ["sun_1"] = "Sonnenstärke",
+  ["sommersprossen"] = "Sommersprossen",
+  ["sommersprossen_1"] = "Sommersprossen Stärke",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:92]:
```suggestion
  ["sommersprossen_1"] = "Sommersprossenstärke",
```

File: [core]/skinchanger/locales/de.lua [hunk 41]
```diff
@@ -0,0 +1,100 @@
...
+  ["chest_hair"] = "Brusthaar",
+  ["chest_hair_1"] = "Brusthaardicke",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:94]:
```suggestion
  ["chest_hair_1"] = "Brusthaarstärke",
```

File: [core]/skinchanger/locales/de.lua [hunk 42]
```diff
@@ -0,0 +1,100 @@
...
+  ["chest_color"] = "Brusthaarfarbe",
+  ["bodyb"] = "Körpermakel",
+  ["bodyb_size"] = "Körpermakel-Dicke",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:97]:
```suggestion
  ["bodyb_size"] = "Körpermakelstärke",
```

File: [core]/skinchanger/locales/de.lua [hunk 43]
```diff
@@ -0,0 +1,100 @@
...
+  ["bodyb_extra"] = "Körperflecken-Effekt",
+  ["bodyb_extra_thickness"] = "Flecken Körper Effekt Dicke",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:99]:
```suggestion
  ["bodyb_extra_thickness"] = "Körperfleckeneffektstärke",
```

File: [core]/skinchanger/locales/de.lua [hunk 44]
```diff
@@ -0,0 +1,100 @@
...
+  ["sun"] = "Sonne",
+  ["sun_1"] = "Sonnenstärke",
+  ["sommersprossen"] = "Sommersprossen",
+  ["sommersprossen_1"] = "Sommersprossen Stärke",
+  ["chest_hair"] = "Brusthaar",
+  ["chest_hair_1"] = "Brusthaardicke",
+  ["chest_color"] = "Brusthaarfarbe",
+  ["bodyb"] = "Körpermakel",
+  ["bodyb_size"] = "Körpermakel-Dicke",
+  ["bodyb_extra"] = "Körperflecken-Effekt",
```
Comment (zRxnx) [inline [core]/skinchanger/locales/de.lua:98]:
```suggestion
  ["bodyb_extra"] = "Körperfleckeneffekt",
```

**Comment by zRxnx** [comment]
> @zRxnx Thanks for the spamm, my phone is on fire now.

pressed the wrong button xd

**Comment by Gellipapa** [comment]
Closed
https://github.com/esx-framework/esx_core/issues/1267
https://github.com/esx-framework/esx_core/issues/1268
https://github.com/esx-framework/esx_core/issues/1270
https://github.com/esx-framework/esx_core/issues/1269

Merged!

**Pull request merged**